lisp/forge-bitbucket.el

```diff
@@ -159,6 +159,13 @@ Callback function CB should accept itself as argument."
                   (setf (alist-get 'comments (car cur)) value)
                   (funcall cb cb)))))
 
+(defun forge--bitbucket-hack-iso8601 (time)
+  "Return TIME - an ISO8601 timestamp - with no more than three second decimals."
+  (when time
```

Comment:
Since this isn't about side-effects use `and` instead of `when`.

Reply:
Done